### ISSUE TITLE
admin.html page

### ISSUE THREAD

**Issue by smellems** [canada-ca/canada-ca.github.io#28]
@Borges-Porteous what is this page for?  Maybe I can help with what you want to do

**Comment by Borges-Porteous** [comment]
@smellems Hey! I set up a GA account for the blog, so I was looking for a way to segment out internal visits from the team using the blog, without having to do an IP based filter, since other government and even TBS employees are part of our audience.

For the page itself, I just added the index of articles so that it wouldn't be blank. The trick now is convincing my team to use it...

If you're familiar with GA, then any other ideas on how to filter out external traffic would be appreciated.

**Comment by smellems** [comment]
@Borges-Porteous Hi!  I did remove the GA code from the HTML when I created the site, good job adding it back.

Another solution I'm thinking would be to use an ad blocker (uBlock) that will block the call to GA, so not count that visiter.

**Comment by Borges-Porteous** [comment]
@smellems I had not even realised there was a GA code for canada-ca before. I made a new tracking code, but I guess we'll just ignore the old one, since I've already shared it with team members.

This uBlock idea sounds interesting, but it runs into one main problem, which is that I'll have to convince my team to install it, not just on their work computers, but home computers and mobiles too, which I'm not sure will happen.

I'll give it a try if my team really can't adjust to using the admin page though. It seems like a good method to do it once and then no need to think about it again.

**Comment by pcwsmith** [comment]
@Borges-Porteous ... about the admin page - I never remember to use it!

**Comment by Borges-Porteous** [comment]
Yeah... I don't really have any other method to do this though... sad...
